docs/fomega/deep-isorecursive/alternatives.md

```diff
@@ -121,19 +119,20 @@ As shown in the end of the previous section there is no big difference to the us
 
 Regardless of the approach, `unwrap` is always used the same way. Even with elimination contexts or without higher kinds at all `unwrap` was used in the same way. The usages of `unwrap` in the Plutus Core codebase have never changed basically.
 
-It also does not matter which approach we choose from the encoding of mutual recursion perspective. This is because for encoding mutual recursion we use [ifix](https://gist.github.com/effectfully/e57d2816c475928a380e5a6b897ad17d) which we have out of the box in the `ifix` approach and which we can get trivially in the head-spine form approach:
+It also does not matter which approach we choose from the encoding of mutual recursion perspective. This is because for encoding mutual recursion we use [ifix](https://github.com/input-output-hk/plutus/blob/a51f2e8bbfad1ce22b3769b1bac9a0b054cadbfd/docs/fomega/mutual-type-level-recursion/MutualData.agda) which we have out of the box in the `ifix` approach and which we can get trivially in the head-spine form approach:
 
 ```
-ifix = \(f :: ((k -> *) -> k -> *) -> k -> *) (a :: k) -> fix f [a]
-iwrap = /\(f :: ((k -> *) -> k -> *) -> k -> *) (a :: k) -> wrap f [a]
+ifix  =  \(f :: (k -> *) -> k -> *) (a :: k) ->                        Spines.fix  f [a]
+iwrap = /\(f :: (k -> *) -> k -> *) (a :: k) -> \(t : f (ifix f) a) -> Spines.wrap f [a] t
```

Comment:
The kind of `f` was wrong in both the cases and I eta-expanded things properly.

Reply:
What are `Spines.fix` and `Spines.wrap`? I think I know, but it's not clear.

Reply:
`fix` and `wrap` from the `Spines` approach. I'm not sure how to communicate the intention properly here, any ideas?

Reply:
Just adding a sentence afterwards saying "where `Spines.fix` and `Spines.wrap` are the corresponding primitives in the head-spine approach" would make it clear.